apiservices/api_configuration_service.go

```diff
@@ -35,88 +35,56 @@ func NewConfigurationApiService() apiserver.ConfigurationAPIServicer {
 
 // GetAdvancedConfiguration - Get Advanced Configuration
 func (s *ConfigurationApiService) GetAdvancedConfiguration(ctx context.Context) (apiserver.ImplResponse, error) {
-	// TODO - update GetAdvancedConfiguration with the required logic for this service method.
-	// Add api_configuration_service.go to the .openapi-generator-ignore to avoid overwriting this service implementation when updating open api generation.
+	advCnf, err := conf.GetAdvancedConfig(ctx)
 
-	//TODO: Uncomment the next line to return response Response(200, AdvancedConfiguration{}) or use other options such as http.Ok ...
-	//return Response(200, AdvancedConfiguration{}), nil
-
-	return apiserver.Response(http.StatusNotImplemented, nil), errors.New("GetAdvancedConfiguration method not implemented")
+	return apiserver.Response(http.StatusOK, advCnf), err
```

Comment:
1) This is already implemented in develop branch. Please rebase.
2) StatusOK even with error?

Reply:
1. Rebased (Merged)
2. Not really, but it wasn't really finished yet. :sweat_smile: